src/main/java/org/jenkinsci/backend/confluence/pageremover/Space.java

```diff
@@ -44,4 +44,18 @@ public static Space find(MimeMessage msg) throws MessagingException {
         }
         return null;
     }
+
+    /**
+     * Finds the space from a mail subject.
+     *
+     * @return null if no space matches
+     */
+    public static Space find(String subject) {
+        for (Space s : SPACES) {
```

Comment:
Is there a possibility we can get this to run under Java 8? I see for loops, and I long for lambdas...